src/main/java/roomescape/admin/controller/AdminRestController.java

```diff
@@ -55,4 +59,24 @@ public ResponseEntity<List<AdminReservationResponse>> getReservationsBySearch(
 
         return ResponseEntity.ok(searchedResponses);
     }
+
+    @GetMapping("/waiting-management")
```

Comment:
아래와 같은 path 만으로도 충분히 표현될 것 같네요~

```suggestion
    @GetMapping("/waitings")
```

Reply:
명사 중심의 리소스가 아니라 기능 중심의 용어인 management가 들어가 있어서, 다소 REST와는 거리가 있는 URI인 것 같다는 생각이 들었습니다.
반면에, 제안 해주신 /waitings에 대한 요청은 "대기 정보 목록을 조회한다"는 의미가 자연스럽게 해석이 되는 것 같아요.

수정하겠습니다!